changelog/february2023/2023-02-07-kubernetes-removed-container-runtimes-docker-and.mdx

```diff
@@ -11,5 +11,5 @@ product: kubernetes
 
 All new pools created in Scaleway Kubernetes clusters now use `Containerd` as their default runtime. 
 
-We recommend you [migrate node pools](/containers/kubernetes/how-to/change-container-runtime-interface/) to a supported CRI. You can find more details about migration policies in our [version support policy page](/containers/kubernetes/reference-content/version-support-policy/).
+We recommend you migrate node pools to a supported CRI. You can find more details about migration policies in our [version support policy page](/containers/kubernetes/reference-content/version-support-policy/).
```

Comment:
```suggestion
```

Hello @tgenaitay I removed the link to the documentation and deleted it. But I can delete the whole sentence.